oak/server/storage/storage_manager.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OAK_SERVER_STORAGE_MANAGER_H_
+#define OAK_SERVER_STORAGE_MANAGER_H_
+
+#include <memory>
+#include <string>
+
+#include "absl/types/span.h"
+#include "oak/server/channel.h"
+
+namespace oak {
+
+class StorageManager {
+ public:
+  StorageManager() {}
+
+  std::unique_ptr<Message> ReadResponseData(uint32_t size) {
+    absl::Span<const char> response_data = operation_response_view_.subspan(0, size);
```

Comment:
This seems like it's aligned more with the previous stream-based channels, rather than the current message-based approach?

Reply:
+1, @michael-kernel-sanders do you plan to fix this in this PR or to follow up later? If the latter, perhaps add a TODO somewhere?

Reply:
Added TODO.